test/fixtures/rules/std_rule_cases/L070.yml

```diff
@@ -0,0 +1,482 @@
+rule: L070
+
+test_pass_known_number_of_result_columns_1:
+  pass_str: |
+    select a, b
+    from t
+    union all
+    select c, d
+    from k
+
+test_fail_unknown_number_of_result_columns_1:
```

Comment:
I agree this test case should fail, but if I read it correctly, the number of result columns is known (1 for the first select, 2 for the second). Should the test case be renamed?